app/app/settings/base.py

```diff
@@ -43,8 +51,18 @@
     "django.middleware.locale.LocaleMiddleware",
     "django.middleware.gzip.GZipMiddleware",
     "django_htmx.middleware.HtmxMiddleware",
+    # AxesMiddleware should be the last middleware in the MIDDLEWARE list.
+    "axes.middleware.AxesMiddleware",
 ]
 
+# Axes configuration
+AXES_FAILURE_LIMIT = config("AXES_FAILURE_LIMIT", default=5, cast=int)
+AXES_COOLOFF_TIME = config("AXES_COOL_OFF_TIME", default=0.01, cast=float)  # in hours
```

Comment:
Can you add these to the documentation? The documentation doesn't have to be extensive, but maybe you could link to the django-axes documentation for each of these